src/wx/wayland.cpp

```diff
@@ -1,11 +1,31 @@
+#define private public
```

Comment:
Please, limit this to being behind the `WX_HAS_BROKEN_WAYLAND_OPENGL` check and do `#undef private` after the include. The include could be limited to `#include <wx/unit/glegl.h>`